amy/templates/reports/membership_trainings_stats.html

```diff
@@ -16,6 +16,7 @@
         <th rowspan="2">Agreement</th>
         <th rowspan="2">Contribution</th>
         <th colspan="3" class="text-center">Instructor training seats (combined public and in-house)</th>
+        <th colspan="3" class="text-center">CLDT seats</th>
```

Comment:
This introduces 3 new columns in the row that's following (lines 23-25 below). Can you duplicate those lines? This way the header displays correctly:

<img width="1083" alt="image" src="https://github.com/user-attachments/assets/45c370c4-81f8-4b49-b056-a3c9bfef93c5">

Reply:
Done!